Assets/Scripts/Background/BeyondBorderResize.cs

```diff
@@ -0,0 +1,118 @@
+            cameraEdge = Camera.main.transform.position.x - (cameraWidth / 2);
+        } else {
+            borderEdge = border.transform.position.x + (borderWidth / 2);
+            cameraEdge = Camera.main.transform.position.x + (cameraWidth / 2);
+        }
+        
+        fillerSprite = GetComponent<SpriteRenderer>();
+
+        // If background + edges combined are already wider than the camera, no need to resize sprite.
+        if ((right && borderEdge < cameraEdge) || (!right && borderEdge > cameraEdge)) {
+            spriteWidth = fillerSprite.size.x;
+            spriteHeight = fillerSprite.size.y;
+
+            // The sizing factor is calculated, with the sprite renderer transformed to fill an empty side of the screen.
+            if (!right) {
+                sizeFactorWidth = Mathf.Abs(borderEdge - cameraEdge) / spriteWidth;
+            } else {
+                sizeFactorWidth = Mathf.Abs(cameraEdge - borderEdge) / spriteWidth;
+            }
+
+            sizeFactorHeight = cameraHeight / spriteHeight;
+            transform.position = new Vector2((borderEdge + (cameraEdge)) / 2, Camera.main.transform.position.y);
+            fillerSprite.size = new Vector2(spriteWidth * sizeFactorWidth / localXScale, spriteHeight * sizeFactorHeight / localXScale);
+        }
+    }
+
+
+    // ChangeSize
+    // This function updates the empty right/left space when the camera size has changed (just like the Awake function).
+    private void ChangeSize()
+    {
+        float newSizeFactorWidth;
+        float newSizeFactorHeight;
+
+        if (!right) {
+            cameraEdge = Camera.main.transform.position.x - (cameraWidth / 2);
+        } else {
+            cameraEdge = Camera.main.transform.position.x + (cameraWidth / 2);
+        }
+        
+        if ((right && borderEdge < cameraEdge) || (!right && borderEdge > cameraEdge)) { 
+            spriteWidth = fillerSprite.size.x;
+            spriteHeight = fillerSprite.size.y;
+            
+            if (!right) {
+                newSizeFactorWidth = Mathf.Abs(borderEdge - cameraEdge) / spriteWidth;
+            } else {
+                newSizeFactorWidth = Mathf.Abs(cameraEdge - borderEdge) / spriteWidth;
+            }
+
+            newSizeFactorHeight = cameraHeight / spriteHeight;
+            transform.position = new Vector2((borderEdge + (cameraEdge)) / 2, Camera.main.transform.position.y);
+            fillerSprite.size = new Vector2(spriteWidth * newSizeFactorWidth / localXScale, spriteHeight * newSizeFactorHeight / localXScale);
+
+            sizeFactorWidth = newSizeFactorWidth;
+            sizeFactorHeight = newSizeFactorHeight;
+        }
+    }
+    
+
+    void Update()
+    {
+        Vector2 newSize;
+
+        cameraHeight = Camera.main.orthographicSize * 2;
+        cameraWidth = cameraHeight * Camera.main.aspect;
+        newSize = new Vector2(cameraWidth, cameraHeight);
+
+        if (currentSize != newSize) {            
+            ChangeSize();

```

Comment:
`ChangeBackgroundFillSize` (or `UpdateBackgroundFillSize`) can be used instead of `ChangeSize`.

Reply:
Got it, "UpdateBackgroundFillSize" added instead.